modules/migrating-azure-dns-zones.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// *
+
+[id="migrating-azure-dns-zones_{context}"]
+= Migrating legacy Azure DNS zones
+
+If you installed an {product-title} version 4.2 or 4.3 cluster on Azure that uses installer-provisioned infrastructure, you must migrate the legacy private DNS zone to the new type of private DNS zone.
+
+For more information about private zones in Azure, see link:https://azure.microsoft.com/en-us/updates/announcing-preview-refresh-for-azure-dns-private-zones-2/[Announcing Preview Refresh for Azure DNS Private Zones].
+
+.Prerequisites
+
+* Obtain the {product-title} installation program.
+
+.Procedure
+
+. Determine your legacy zones:
++
+----
+$ ./openshift-install migrate azure-privatedns-eligible
+
+INFO[0000] Credentials loaded from file "<path_to>/ServicePrincipal.json"
+INFO[0002] legacy zone=<legacy_zone> resourceGroup=<resource_group_name>
```

Comment:
The [0000] and [0002] are removed now, so only INFO shows.

Reply:
Fixed! Thank you!